### DISCUSSION TITLE
How to disable wezterm-gui.exe 's logging?

### DISCUSSION THREAD

**Discussion by Glomzzz** [wezterm/wezterm#5924] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

wezterm 20240730-082727-56a27e93

### Ask your question!

How to disable wezterm-gui.exe 's logging?

I'm running on Windows11, when i open a terminator by lauching wezterm-gui.exe, the log file will be created in the current work dir.

But i dont want any log file, I want to disable it or move it to a sepecific dir

**Comment by Glomzzz** [comment]
Anyone?

**Comment by Glomzzz** [comment, marked as answer]
When I set `front_end = "WebGpu"`, it will create a log file anytime i lauch wezterm-gui.exe

**Comment by Sets3n** [comment]
Is there a solution to this problem? I'm encountering it too now.
